package.json

```diff
@@ -22,8 +22,11 @@
   },
   "author": "The Betaflight open source project.",
   "license": "GPL-3.0",
-  "dependencies": {},
+  "dependencies": {
+    "gulp-appdmg": "^1.0.3"
```

Comment:
We can't have this dependency in here - this will keep `npm install` from working on non-macosx. (Also, it is not a runtime dependency at all.)

The way that `platform-dependent-modules` works, and the consequence that `package.json` is modified every time a platform dependent module is installed, makes it hard and tedious to work with. I have tried some alternative, but wasn't able to find one that actually works better in this respect.

Reply:
That's odd, didn't move that dependency as I know it belongs in the platform dependent modules. I think the clean `npm install` did this automatically. I will check if I can reproduce it, if this is the case then this can happen more often when someone does a clean modules install.

Reply:
Yes, unfortunately `platform-dependent-modules` does this on `npm install`, making it a pain to develop with it.

Reply:
OK, saves me time to reproduce this. I will change it back and take extra care with future `npm install` clean installs